src/services/Store/store/account.slice.ts

```diff
@@ -71,3 +81,26 @@ export const getWalletAccountsOnDefaultNetwork = createSelector(
   getAccounts,
   pipe(reject(propEq('wallet', WalletId.VIEW_ONLY)), filter(propEq('networkId', DEFAULT_NETWORK)))
 );
+
+/**
+ * Actions
+ */
+export const addAccounts = createAction<IAccount[]>(`${slice.name}/addAccounts`);
+
+/**
+ * Sagas
+ */
+export function* accountsSaga() {
+  yield takeLatest(addAccounts.type, handleAddAccounts);
+}
+
+export function* handleAddAccounts({ payload }: PayloadAction<IAccount[]>) {
+  const isDemoMode = yield select(getIsDemoMode);
+  if (isDemoMode) {
+    yield put(slice.actions.resetAndCreateMany(payload));
+    yield put(addAccountsToFavorites(payload.map(({ uuid }) => uuid)));
```

Comment:
Can we add a comment that this disables demo mode or make it more explicit in some way?

Reply:
This one hasn't been solved either.